coding_schemes/Cash_Modality_Yesno.json

```diff
@@ -89,14 +89,6 @@
       "CodeID": "code-564016a9184b",
       "CodeType": "Normal"
     },
-    {
```

Comment:
We can only do this if there is no data coded with this code - I've just confirmed this, please please can you explicitly state it on future PRs that delete codes?

Reply:
Thank you @lukechurch . This is noted.